model/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReportable.scala

```diff
@@ -12,11 +11,11 @@ trait ErrorReportable {
     throwable match {
       case gjre: GoogleJsonResponseException =>
         val statusCode = StatusCodes.getForKey(gjre.getStatusCode)
-        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre), Seq.empty, Option(gjre.getClass))(errorReportSource)
+        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre)(errorReportSource), Seq.empty, Option(gjre.getClass))(errorReportSource)
```

Comment:
I thought the whole point of this class was to define a source other than the one defined implicit-y. Before the code was moved it was using the local errorReportSource for the top ErrorReport but the implicit source for the causes. So, it might look like the top is from "google" and the causes are from "rawls." This change makes the whole stack come from "google" (or whatever errorReportSource is set to be).